### PR TITLE
PTX-20723 - Stork Cloud Jobs fix

### PR DESCRIPTION
**Pull request by gejain**
**What type of PR is this?**
>integration-test

**What this PR does / why we need it**:
This PR is needed for stork cloud jobs to run migration tests on AKS/GKE

Test Run on AKS - https://jenkins.pwx.dev.purestorage.com/job/Stork/view/Stork%20Cloud%20Jobs/job/stork-aks-migration/1/

### CODE REVIEW

**Review by Rohit-PX — APPROVED** [review]

File: test/integration_test/common_test.go [hunk 1]
```diff
@@ -304,8 +304,8 @@ func setup() error {
 		return fmt.Errorf("while PX service to LoadBalancer, setting kubeconfig to destination failed %v", err)
 	}
 
-	// On destination cluster, change PX service to type LoadBalancer if it is an EKS cluster
-	if volumeDriverName == storkdriver.PortworxDriverName && IsEks() {
+	// On destination cluster, change PX service to type LoadBalancer if it is an EKS/Aks/GKE cluster
```
Comment (pp511) [inline test/integration_test/common_test.go:307]: nit.. AKS

File: test/integration_test/common_test.go [hunk 2]
```diff
@@ -1563,11 +1563,11 @@ func changePxServiceToLoadBalancer(internalLB bool) error {
 	if volumeDriverName == storkdriver.PortworxDriverName {
 		stc, err := operator.Instance().ListStorageClusters(pxNamespace)
 		if err != nil {
-			return fmt.Errorf("failed to list PX storage cluster on EKS: %v", err)
+			return fmt.Errorf("failed to list PX storage cluster on Eks/Aks/Gke: %v", err)
```
Comment (pp511) [inline test/integration_test/common_test.go:1566]: Nit..EKS/AKS/GKE

**Review by pp511 — APPROVED** [review]
lgtm

**Review by diptiranjanpx — APPROVED** [review]

File: test/integration_test/common_test.go (hunk 1 above)
Comment (gejain) [reply test/integration_test/common_test.go]: Done, Thanks!

File: test/integration_test/common_test.go (hunk 2 above)
Comment (gejain) [reply test/integration_test/common_test.go]: Done, Thanks!

**Pull request merged**